PROJECT

```diff
@@ -0,0 +1,7 @@
+---
+projectName: triggers
+domain: triggers.shipwright-io
```

Comment:
nit - domain should be `triggers.shipright.io`. I imagine this has a downstream impact on the generated CRDs.

Reply:
Thanks, I haven't noticed the typo in the `domain` attribute. Fixing it.